roles/edpm_libvirt/molecule-requirements.txt

```diff
@@ -0,0 +1,8 @@
+ansible-core<2.12.0
```

Comment:
If possible we should centralize requirements in common file we have for that purpose.

Reply:
we cant in this case as i noted in the libvirt PR.
there is a depenciy issue with the vagrant molecule plugin.
https://github.com/openstack-k8s-operators/edpm-ansible/pull/215#discussion_r1263630455